test/verify/check-packagekit

```diff
@@ -239,6 +242,60 @@ class TestUpdates(NoSubManCase):
         self.assertEqual(b.attr(self.update_icon, "class"), "fa fa-check-circle-o")
         b.wait_text(self.update_text, "System is up to date")
 
+    def testRequireReboot(self):
+        b = self.browser
+        m = self.machine
+
+        self.createPackage("cockpit-testing-package", "1", "1", install=True)
+        self.createPackage("cockpit-testing-package", "1", "2", changes="very important changes")
```

Comment:
This doesn't do anything special. How does that trigger tracer to require a reboot?

Reply:
I don't really know how to trigger a reboot option apart from just hardcoding some dummy package. Tracer has a list of packages which require a reboot (like kernel or systemd), but creating a package in test for that would not work. So I created a package called cockpit-testing-package and hardcoded that for that package it would act like a reboot is needed. It's an ugly solution, but I cannot think of better.

Reply:
Fair enough, but where is that actually wired up? I only see "cockpit-testing-package" in the test checks, but not any /etc/tracer.config - like setup that would tell tracer to consider that as a package which needs rebooting? (Also, if you do that, please give it a more specific name, like "pkg-needs-reboot" or so).

Why would it not work to create a fake package that requires reboot? E.g. the test could totally create a (mostly empty) kernel-9999.rpm package and upgrade to that, as long as the test does not *actually* reboot (that would fail, of course), but just makes sure that the "this requires rebooting" state is reached. But perhaps there is some other package in tracer's list which is not crucial for our testing?